sklearn/multiclass.py

```diff
@@ -165,6 +165,18 @@ def predict_proba(self, X):
         return np.repeat([np.hstack([1 - self.y_, self.y_])], _num_samples(X), axis=0)
 
 
+def _estimators_has(attr):
```

Comment:
Shall we keep it in multiclass? It seems that it could be a utility for more meta-estimator indeed.
I could imagine putting it in the `metaestimators.py` or maybe in a `sklearn/utils/delegation.py`?

Reply:
There are minor differences across different estimators and ensembles. I'm happy with it being local...

Reply:
OK then it seems that we can go forward and merge